src/validation.cpp

```diff
@@ -1171,7 +1170,7 @@ CAmount GetBlockSubsidyInner(int nPrevBits, int nPrevHeight, const Consensus::Pa
 CAmount GetBlockSubsidy(const CBlockIndex* const pindex, const Consensus::Params& consensusParams)
 {
     if (pindex->pprev == nullptr) return Params().GenesisBlock().vtx[0]->GetValueOut();
-    bool isV20Active = llmq::utils::IsV20Active(pindex->pprev);
+    const bool isV20Active{DeploymentActiveAt(*pindex, consensusParams, Consensus::DEPLOYMENT_V20)};
```

Comment:
unrelated; I hate that `DeploymentActiveAt` wants a ref while `DeploymentActiveAfter` wants a ptr

Reply:
```suggestion
    const bool isV20Active{DeploymentActiveAfter(pindex->pprev, consensusParams, Consensus::DEPLOYMENT_V20)};
```